react-native-libraries.json

```diff
@@ -11581,6 +11581,28 @@
     "web": true,
     "expoGo": true
   },
+  {
+    "githubUrl": "https://github.com/Aris-ngoy/react-native-irano",
+    "npmPkg": "react-native-irano",
+    "examples": [
+      "https://github.com/Aris-ngoy/react-native-irano/tree/main/example"
+    ],
+    "images": [
+      "https://github.com/Aris-ngoy/react-native-irano/blob/main/assets/1.png?raw=true",
+      "https://github.com/Aris-ngoy/react-native-irano/blob/main/assets/2.png?raw=true",
+      "https://github.com/Aris-ngoy/react-native-irano/blob/main/assets/3.png?raw=true",
+      "https://github.com/Aris-ngoy/react-native-irano/blob/main/assets/4.png?raw=true"
+    ],
+    "ios": true,
+    "android": true,
+    "web": true,
+    "expoGo": true,
+    "windows": true,
+    "macos": true,
+    "tvos": true,
+    "visionos": true,
```

Comment:
We require a dedicated examples for out-of-tree platforms, and the example located in repository only uses Expo.

https://github.com/react-native-community/directory?tab=readme-ov-file#%EF%B8%8F-out-of-tree-platforms